reference/OpenAPI.json

```diff
@@ -14244,37 +14243,75 @@
             "type": "object",
-            "description": "Configure parameters returned in the response.",
             "properties": {
-              "expand": {
-                "type": "array",
-                "description": "Expand array lets you configure params included in the response. Depending on the strings included in the array, the response will contain different details. \n\n| **Expand Option** | **Response Body** |\n|:---|:---|\n| [\"order\"] | - Same response as fallback response (without an options object).<br>- Order data with calculated discounts are listed in each child redeemable object.<br>- Metadata not included for each discount type. |\n| [\"redeemable\"] | Expands redeemable objects by including `metadata` for each discount type. |\n| [\"order\", \"redeemable\"] | - Order data with calculated discounts are listed in each child redeemable object.<br>- Includes `metadata` for each discount type. |\n| [\"category\"] | - Returns an expanded `categories` object, showing details about the category. |",
-                "items": {
-                  "type": "string",
-                  "enum": [
-                    "order",
-                    "redemption",
-                    "redeemable",
-                    "category"
-                  ]
+              "options": {
+                "type": "object",
+                "description": "Configure parameters returned in the response.",
+                "properties": {
+                  "expand": {
+                    "type": "array",
+                    "description": "Expand array lets you configure params included in the response. Depending on the strings included in the array, the response will contain different details. \n\n| **Expand Option** | **Response Body** |\n|:---|:---|\n| [\"order\"] | - Same response as fallback response (without an options object).<br>- Order data with calculated discounts are listed in each child redeemable object.<br>- Metadata not included for each discount type. |\n| [\"redeemable\"] | Expands redeemable objects by including `metadata` for each discount type. |\n| [\"order\", \"redeemable\"] | - Order data with calculated discounts are listed in each child redeemable object.<br>- Includes `metadata` for each discount type. |\n| [\"category\"] | - Returns an expanded `categories` object, showing details about the category. |",
+                    "items": {
+                      "type": "string",
+                      "enum": [
+                        "order",
+                        "redemption",
+                        "redeemable",
+                        "category"
+                      ]
+                    }
+                  }
                 }
               }
             }
+          },
+          {
+            "$ref": "#/components/schemas/StackableValidateRedeemRequestBase"
           }
-        },
-        "required": [
-          "redeemables"
         ]
       },
       "RedemptionsRedeemRequestBody": {
```

Comment:
To nie powinno się nazywać `Validations RedeemRequestBody` skoro zmieniasz title?

Reply:
changed, thanks